WordPress/Classes/ViewRelated/Comments/CommentsViewController.m

```diff
@@ -446,6 +450,31 @@ - (void)syncContentEnded
     [self refreshPullToRefresh];
 }
 
+#pragma mark - Sync Notification Methods
+
+- (void)registerForSyncNotification
+{
+    [[NSNotificationCenter defaultCenter] addObserver:self
+                                             selector:@selector(syncNotificationReceived:)
+                                                 name:CommentServiceFinishedSyncingNotification
+                                               object:nil];
+}
+
+- (void)unregisterForSyncNotification
+{
+    [[NSNotificationCenter defaultCenter] removeObserver:self name:CommentServiceFinishedSyncingNotification object:nil];
+}
+
+- (void)syncNotificationReceived:(NSNotification *)notification
+{
+    NSDictionary *notificationInfo = notification.userInfo;
+    NSManagedObjectID *blogObjectID = notificationInfo[@"objectID"];
+    if ([blogObjectID isEqual:self.blog.objectID])
+    {
+        [self.tableViewHandler refreshTableView];
```

Comment:
Yup, it's still working fine without it. Not sure why I thought I needed it… 🤔 but the code is a touch simpler now 👍